pkg/pgmodel/lreader/labels_reader.go

```diff
@@ -59,7 +59,7 @@ EXISTS(
 )
 `
 	getLabelNamesForTenant  = `SELECT array_agg(distinct l.key) FROM _prom_catalog.label l WHERE %s`
-	getLabelValuesForTenant = `SELECT array_agg(distinct a.value) FROM (SELECT * FROM _prom_catalog.label l WHERE %s) a WHERE a.key = %s`
+	getLabelValuesForTenant = `SELECT array_agg(distinct value) FROM _prom_catalog.label l WHERE key = $1 AND %s`
```

Comment:
```suggestion
	getLabelValuesForTenant = `SELECT array_agg(distinct value) FROM _prom_catalog.label l WHERE key = $%d AND %s`
```

Reply:
Why `%d` here, when it will always be `1`?